.github/workflows/ci.yml

```diff
@@ -110,37 +110,35 @@ jobs:
           sudo apt-get update
           sudo apt-get install --no-install-recommends -y tree libasound2-dev libglib2.0-dev libxcb-shape0-dev libxcb-xfixes0-dev \
             libcairo-dev libgtk2.0-dev libsoup2.4-dev libgtk-3-dev libwebkit2gtk-4.0-dev xorg-dev ninja-build libxcb-render0-dev
+      - name: Install protoc
```

Comment:
Is this still necessary since this doesn't do a workspace build?

Reply:
I don't think it is, will remove.